rosdep/python.yaml

```diff
@@ -5240,6 +5244,9 @@ python3-boto3:
   debian: [python3-boto3]
   fedora: [python3-boto3]
   opensuse: [python3-boto3]
+  rhel:
+    '*': ['python%{python3_pkgversion}-boto3']
```

Comment:
python3-boto3 in EPEL 8 http://rpmfind.net/linux/RPM/epel/8/x86_64/Packages/p/python3-boto3-1.10.21-1.el8.noarch.html